oemoflex/usecase.py

```diff
@@ -0,0 +1,57 @@
+import oemof
+import oemof.tabular
+from oemoflex.helpers import make_dirs
+
+
+class Usecase:
```

Comment:
```suggestion
class Scenario:
```

Reply:
Use case is quite project specific. Scenario can be understood in many ways. Maybe, ScenarioRun or ModelChain would be better because they are more concrete?